src/pages/Chapter.jsx

```diff
@@ -0,0 +1,16 @@
+import { Link, useLocation } from "react-router-dom";
+import "../styles/Chapter.css";
+
+function Chapter() {
+  const location = useLocation();
+  const { chapterName, chapterUrl } = location.state;
+  return (
+    <section className="ChapterComponent">
+      <Link to="/list">Retourner à la liste</Link>
```

Comment:
It's a bit disturbing for the user to go back to the book list, and not the book detail page

Reply:
You right ! I made a change on this with the useNavigate an navigate(-1) instead